pyfar/dsp/filter.py

```diff
@@ -679,3 +680,285 @@ def _shelve(signal, frequency, gain, order, shelve_type, sampling_rate, kind):
         # return the filtered signal
         signal_filt = filt.process(signal)
         return signal_filt
+
+
+def center_frequencies_fractional_octaves(
+        num_fractions=1, frequency_range=(20, 20e3)):
```

Comment:
`num_fractions` sounds a bit un-intuitive to me. How about calling this `bandwidth`? In any case, the docstring should have an example for clarification (also for the other functions, where this is used).

Reply:
imho. bandwidth is misleading here, since we're dividing octave band into a number of it's fractions. The bandwidth results from this, but is not really what one is specifying with the parameter. I'll update the docstring.